src/assembly/package.xml

```diff
@@ -56,6 +59,16 @@
             <includes>
                 <include>io.confluent:kafka-avro-serializer</include>
                 <include>io.confluent:kafka-schema-registry-client</include>
+                <include>io.confluent:copycat-avro-converter</include>
+            </includes>
+        </dependencySet>
+        <dependencySet>
+            <outputDirectory>share/java/json-serializer</outputDirectory>
+            <useProjectArtifact>true</useProjectArtifact>
+            <scope>runtime</scope>
+            <useTransitiveFiltering>true</useTransitiveFiltering>
+            <includes>
+                <include>io.confluent:kafka-json-serializer</include>
```

Comment:
We actually added a JSON serializer to schema registry before, I just noticed this was missing as I was adding the converter. I can move it out into a separate patch if you prefer.